lib/block_view_types.go

```diff
@@ -682,7 +682,8 @@ const (
 	OperationTypeStakeDistributionRestake      OperationType = 49
 	OperationTypeStakeDistributionPayToBalance OperationType = 50
 	OperationTypeSetValidatorLastActiveAtEpoch OperationType = 51
-	// NEXT_TAG = 52
+	OperationTypeAtomicTxnsWrapper             OperationType = 52
+	// NEXT_TAG = 53
```

Comment:
This also reminds me, please update `GetEncoderForTxType` to return some value for `TxnTypeAtomicTxnsWrapper`. Without this, state syncer will panic and crash on the first atomic txn it sees. It can simply be an empty struct for now - although I'd probably just make it the following struct:
```
type AtomicTxnsWrapper struct {
  InnerTxnsTransactionMetadata []TransactionMetadata 
}
```
This way we can compute txindex metadata properly for it and use it in state syncer. And then a simple update to `ComputeTransactionMetadata` that calls itself for each inner txn would get you want you want.